des_y6utils/mdet.py

```diff
@@ -7,6 +7,62 @@
 import healsparse
 
 
+def add_extinction_correction_columns(fold, fnew, fdust):
```

Comment:
I think we'll want to convert this function to act derectly on some data and output new data.

Reply:
Is this not optimal yet? We could have a line that deletes the old file, too.

Reply:
Pushing to and from disk is expensive and should be up to the user. I'll push the change.